.ci/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e -x
+
+if [[ ${BIGCHAINDB_CI_ABCI} == 'enable' ]]; then
```

Comment:
shouldn't you get an `unbound` variable error when this script is executed for other matrices e.g. `3.5`? Same for other instances? Why not make it global?

Reply:
ignore my comment, if it is not exposed it will be treated with `''`

Reply:
Undeclared vairables seem to be replaced with empty string, [here](https://travis-ci.org/bigchaindb/bigchaindb/jobs/364526325#L1045)